package.json

```diff
@@ -97,6 +98,7 @@
     "x-xss-protection": "^1.0.0"
   },
   "dependencies": {
+    "@types/express": "^4.11.0",
```

Comment:
This is not needed. We do not depend on Express.

Reply:
I'll work on removing it.